python/ray/serve/_private/autoscaling_state.py

```diff
@@ -161,13 +163,17 @@ def record_request_metrics_for_replica(
     ) -> None:
         """Records average number of ongoing requests at a replica."""
 
+        if not replica_metric_report:
```

Comment:
in what situation will replica_metric_report evaluate to false, the function type hint suggests that replica_metric_report cannot be None?

Reply:
Removed